ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.tsx

```diff
@@ -717,6 +678,78 @@ export const policyConfigurationDescriptor: Descriptor[] = [
         placeholder: '22',
         canBooleanLogic: true,
     },
+    {
+        label: 'Port Exposure',
+        name: 'Port Exposure Method',
+        shortName: 'Port exposure method',
+        negatedName: 'Port exposure method is not',
+        category: policyCriteriaCategories.NETWORKING,
+        type: 'select',
+        options: Object.keys(portExposureLabels)
+            .filter((key) => key !== 'INTERNAL')
+            .map((key) => ({
+                label: portExposureLabels[key],
+                value: key,
+            })),
+        canBooleanLogic: true,
+    },
+    {
+        label: 'Network baselining enabled',
+        name: 'Unexpected Network Flow Detected',
+        shortName: 'Unexpected network flow detected',
+        longName: 'Network baselining status',
+        category: policyCriteriaCategories.NETWORKING,
+        type: 'radioGroup',
+        radioButtons: [
+            { text: 'Unexpected network flow', value: true },
+            { text: 'Expected network flow', value: false },
+        ],
+        defaultValue: false,
+        reverse: false,
+        canBooleanLogic: false,
```

Comment:
oh does this one not need a feature flag?

Reply:
Good question. Descriptor at end of file was not moved into the array when feature flag was removed.